app-server/doky-front/src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Logo = () => {
+  return (
+    <>
```

Comment:
### Unnecessary wrapping in <></> (NEED TO FIX)

You can just write:

```js
const Logo = () => {
  return <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>;
};
```

There is no reason to create additional node. It will not appears in DOM but will live (I suspect) in [virtual DOM](https://www.geeksforgeeks.org/reactjs-virtual-dom/). Not a big waste but let's clean up this.

`<></>` - called fragment and similiar (but not the same) to [fragment used in DOM API.](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment)

### Use short from of arrow function (OPTIONAL)

```js
const Logo = () =>
  <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>;
```

Optionally you can wrap returned jsx exression into brackets.

```js
const Logo = () => (
  <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>
);
```

Learn more: https://javascript.info/arrow-functions-basics

However, you can still use long form of arrow function with `return` keyword and curly braces around body. 

### Optimization for static jsx (NEED TO FIX)

Probably there is no reason to create a function that returns same value (***not reference**) every time. This leads to unnecessary checks during [reconcilation](https://www.geeksforgeeks.org/reactjs-reconciliation/) 

You can just use constant

```js
const LOGO = <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>;
```

Then use it like this

```js
<div>{LOGO}</div>
```

If you don't like such king of syntax and prefer still to use component you can use memoization. [React provide build-in utility for this](https://react.dev/reference/react/memo). 

```js
import { memo } from 'react';

const Logo = () =>
  <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>;

export default memo(Logo);
```

Please not that memoized function (component) still need to be named function. Otherwise react will warn you in console in dev mode.

```js
import { memo } from 'react';

// WILL LEAD TO WARNING!!!
const Logo = memo(() =>
  <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>
);

export default Logo;
```

Lear more:
 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name
 * https://medium.com/@stevemao/do-not-use-anonymous-functions-to-construct-react-functional-components-c5408ec8f4c7